lametro/models/cms.py

```diff
@@ -18,3 +24,38 @@ class AboutPage(Page):
     content_panels = Page.content_panels + [
         FieldPanel("body"),
     ]
+
+
+class Alert(models.Model):
+    include_in_dump = True
+
+    def __str__(self):
+        return f"{self.type} Alert - {self.description[:50]}"
+
+    TYPE_CHOICES = [
+        ("primary", "Primary"),
+        ("secondary", "Secondary"),
+        ("success", "Success"),
+        ("danger", "Danger"),
+        ("warning", "Warning"),
+        ("info", "Info"),
+    ]
+
+    description = models.TextField(null=True, blank=True)
+    type = models.CharField(max_length=255, choices=TYPE_CHOICES)
+
+    panels = [
+        FieldPanel(
+            "type",
+            help_text="Select a style for your alert.",
+        ),
+        FieldPanel("description", widget=MarkdownTextarea),
+    ]
+
+    def content(self):
```

Comment:
The ModelAdmin can render custom model methods in the list display. This method renders the alert description and adds a styled label for the alert type.


<img width="947" alt="Screenshot 2024-11-22 at 10 04 39 AM" src="https://github.com/user-attachments/assets/9364d0cb-15b2-42ad-84f0-b67be51affd5">